### DISCUSSION TITLE
Created workspace with custom env from react-env, now can't start component

### DISCUSSION THREAD

**Discussion by fabio-silva** [teambit/bit#7430] [Help]
Hey there.

As per the docs, I did:
```
bit new react my-workspace --env teambit.react/react-env --default-scope customOrg.customScope
```

The, created a component in a different scope:
```
bit create react pages/base --scope customOrg.design
```

When I do `bit start pages/base`, I get:
```
teambit.harmony/aspect-loader, error: unable to load the extension "customOrg.design/pages/base", due to an error "Unexpected token 'export'", please use the '--log=error' flag for the full error.
```

Am I missing something? :/

**Comment by ranm8** [comment]
Not sure what was the issue. But please just use:

You can initialize Bit on an existing workspace or a new directory using the following command:

```
bit init
```

To create a new directory, run the following command:

```
bit new basic my-workspace
```

You can also start from our React starter to start from example components:

```
bit new react my-project --env bitdev.react/react-env --default-scope my-org.my-project
```
